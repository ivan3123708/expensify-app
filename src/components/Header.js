import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink to="/" activeClassName="selected" exact>Dashboard</NavLink>
        <NavLink to="/create" activeClassName="selected">Create Expense</NavLink>
    </header>
);

export default Header;