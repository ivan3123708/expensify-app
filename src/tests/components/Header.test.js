import React from 'react';
/*
React Shallow Renderer - simpler version of Enzyme

import ReactShallowRenderer from 'react-test-renderer/shallow';
*/
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('Render Header component', () => {
  /*
  React Shallow Renderer example usage code

  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
  */

  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});