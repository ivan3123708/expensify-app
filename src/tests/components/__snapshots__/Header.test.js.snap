// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header component 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="selected"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="selected"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="selected"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
